docs/powerpoint/insert-slides-into-presentation.md

```diff
@@ -0,0 +1,183 @@
+
+            const selectedSlideID = await getSelectedSlideID();
+
+            context.presentation.insertSlidesFromBase64(chosenFileBase64, {
+                formatting: "UseDestinationTheme",
+                targetSlideId: selectedSlideID + "#"
+            });
+
+            await context.sync();
+        });
+    }
+    ```
+
+### Selecting which slides to insert
+
+You can also use the [InsertSlideOptions](/javascript/api/powerpoint/powerpoint.insertslideoptions) parameter to control which slides from the source presentation are inserted. You do this by assigning an array of the source presentation's slide IDs to the `sourceSlideIds` property. The following is an example that inserts four slides. Note that each string in the array must follow one or another of the patterns used for the `targetSlideId` property.
+
+```javascript
+async function insertAfterSelectedSlide() {
+    await PowerPoint.run(async function(context) {
+        const selectedSlideID = await getSelectedSlideID();
+        context.presentation.insertSlidesFromBase64(chosenFileBase64, {
+            formatting: "UseDestinationTheme",
+            targetSlideId: selectedSlideID + "#",
+            sourceSlideIds: ["267#763315295", "256#", "#926310875", "1270#"]
+        });
+
+        await context.sync();
+    });
+}
+```
+
+> [!NOTE]
+> The slides will be inserted in the same relative order in which they appear in the source presentation, regardless of the order in which they appear in the array.
+
+There is no practical way that users can discover the ID or creation ID of a slide in the source presentation. Also, because *a slide can be manually copied from one presentation to another* any way, so you should usually avoid designing the add-in in such a way that the user must have the source presentation open in addition to the target presentation (where the add-in is running). For these reasons, you can really only use the `sourceSlideIds` property when either you know the source IDs at coding time or your add-in can retrieve them at runtime from some data source. Because users cannot be expected to memorize slide IDs, you also need a way to enable the user to select slides, perhaps by title or by an image, and then correlate each title or image with the slide's ID.
+
+Accordingly, the `sourceSlideIds` property is primarily used in presentation template scenarios: There is a small number of known presentations which serve as pools of slides that can be inserted by using the add-in. In such a scenario, either you or the customer must create and maintain a data source that correlates a selection criterion (such as titles or images) with slide IDs.
+
+## Delete slides
+
+You can delete a slide by getting a reference to the [Slide](/javascript/api/powerpoint/powerpoint.slide) object that represents the slide and call the `Slide.delete` method. The following is an example in which the 4th slide (with zero-based index "3") is deleted.
```

Comment:
Can we move the "zero-based" note into a code comment? I think that would help the code stand alone better.